File: app/src/main/java/com/github/willjgriff/ethereumwallet/di/ControllerScope.java
package com.github.willjgriff.ethereumwallet.di;

import java.lang.annotation.Documented;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

import javax.inject.Scope;

/**
 * Created by Will on 29/01/2017.
 */
@Documented
@Scope
@Retention(RetentionPolicy.RUNTIME)
public @interface ControllerScope {
}

File: app/src/main/kotlin/com/github/wiljgriff/ethereumwallet/di/ControllerScope.kt
package com.github.wiljgriff.ethereumwallet.di

import java.lang.annotation.Documented
import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy
import javax.inject.Scope

/**
 * Created by williamgriffiths on 17/04/2017.
 */
@Documented
@Scope
@Retention(RetentionPolicy.RUNTIME)
annotation class ControllerScope